assets/python/ml/pyml:model:adaboosted_trees_regression:sklearn.pyi

```diff
@@ -1,19 +1,17 @@
-# ----------------------------------------------------------------- #
-#                                                                   #
-#   Workflow unit for adaboosted trees regression with              #
-#   Scikit-Learn. Parameters for the estimator and ensemble         #
-#   are derived from sklearn defaults.                              #
-#                                                                   #
-#   When then workflow is in Training mode, the model is trained    #
-#   and then it is saved, along with the RMSE and some              #
-#   predictions made using the training data (e.g. for use in a     #
-#   parity plot or calculation of other error metrics).             #
-#                                                                   #
-#   When the workflow is run in Predict mode, the model   is        #
-#   loaded, predictions are made, they are un-transformed using     #
-#   the trained scaler from the training run, and they are          #
-#   written to a filed named "predictions.csv"                      #
-# ----------------------------------------------------------------- #
+# ------------------------------------------------------------ #
```

Comment:
Let's move all auto-generated files into a subfolder with the corresponding name: i.e. "autogenerated"

Reply:
This will break our unit tests (which assume that all of the .pyi files are in the asset folders, per the pattern in the rest of the package), will require an additional change to `asset.js` and `tree.js`, and will make future work to automatically generate `assets.js` and `tree.js` for the PythonML work more challenging. 

If it's necessary to organize the files in this manner, I can do it, but this will require more work beyond just moving the files into a subfolder.

Reply:
Let's move the generator scripts then to isolate them

Reply:
Moved